tf2_py/src/tf2_py.cpp

```diff
@@ -129,14 +129,30 @@ static PyObject *transform_converter(const geometry_msgs::TransformStamped* tran
   return pinst;
 }
 
-static int rostime_converter(PyObject *obj, builtin_interfaces::msg::Time *rt)
+static int rostime_converter(PyObject *obj, tf2::TimePoint *rt)
 {
   PyObject *tsr = PyObject_CallMethod(obj, (char*)"to_sec", NULL);
   if (tsr == NULL) {
     PyErr_SetString(PyExc_TypeError, "time must have a to_sec method, e.g. rospy.Time or rospy.Duration");
     return 0;
   } else {
-    (*rt).fromSec(PyFloat_AsDouble(tsr));
+    *rt = tf2::timeFromSec(PyFloat_AsDouble(tsr));
+    Py_DECREF(tsr);
+    return 1;
+  }
+}
+
+static int rostime_converter_blt(PyObject *obj, builtin_interfaces::msg::Time *rt)
```

Comment:
Any other thoughts on this dependency? I think it will be okay for the sake of "always reuse code'.